IBAnimatable/PlaceholderDesignable.swift

```diff
@@ -18,7 +18,8 @@ public extension PlaceholderDesignable where Self: UITextField {
   var placeholderText: String? { get { return "" } set {} }
 
   public func configurePlaceholderColor() {
-    if let placeholderColor = placeholderColor, let placeholder = placeholder {
+    let text = placeholderText ?? placeholder
```

Comment:
`placeholderText` will always return an empty String, so if a user sets `Placeholder` in Interface Builder, the latter will never be considered. See [#499](https://github.com/IBAnimatable/IBAnimatable/pull/499)